pages/consent.tsx

```diff
@@ -0,0 +1,40 @@
+import { FC } from 'react'
+import { GetStaticProps } from 'next'
+import { useTranslation } from 'next-i18next'
+import { serverSideTranslations } from 'next-i18next/serverSideTranslations'
+import Layout from '../components/Layout'
+import LinkButton from '../components/LinkButton'
+
+const Consent: FC = () => {
+  const { t } = useTranslation('consent')
+
+  return (
+    <Layout
+      meta={t('common:meta', { returnObjects: true })}
+      header={t('common:header', { returnObjects: true })}
+      footer={t('common:footer', { returnObjects: true })}
+    >
+      <h1 className="mb-4">{t('header')}</h1>
+      <h2 className="my-14">{t('description')}</h2>
+      <div className="flex justify-center flex-wrap text-xl gap-4">
+        <div id="yesButton">
```

Comment:
id naming convention should be kebab case `yes-button` in this case. We have currently a mix of pascalCase and kebab-case in this project. We can refactor the whole project in another PR to be consistent.